providers/windows/process_windows.go

```diff
@@ -46,6 +46,12 @@ func (s windowsSystem) Processes() (procs []types.Process, err error) {
 	procs = make([]types.Process, 0, len(pids))
 	var proc types.Process
 	for _, pid := range pids {
+		if pid == 0 || pid == 4 {
```

Comment:
I was wondering if these were always 0 and 4 and found 
https://stackoverflow.com/questions/2643902/are-the-system-and-system-idle-process-pids-constant-on-windows. It seems the concensus is yes.